lib/data_exporter.rb

```diff
@@ -13,6 +13,10 @@ def export_csv_from_living_in_europe
     export_csv(living_in_europe_subscriber_lists)
   end
 
+  def export_csv_from_slugs(slugs)
```

Comment:
I wonder if it's worth, just in case, having a similar `export_csv_from_slugs_at` method for getting results at a particular date - it wouldn't surprise me if we need that in the future.